index.html

```diff
@@ -53,53 +56,58 @@ <h1 class="center-text heading-text ts">Welcome to</h1>
     <!-- story summary AND image box-->
     <div class="story center-text">
       <div class="article-1">
-      <div>
-      <p class="ts subheading-text">Serving Guanajuato, Mexico<br>street food</p>
-      <p class=" story content-text">
-        Family owned and operated business that started in 2002. Specialized in
-        Guanajuato, Mexico home style cooking. Incorporating home taught dishes
-        with a spin. welcoming customers with a freindly smile
-      </p>
-      </div>
-      <img src="img/tacos-truck-pic.jpeg" />
+        <div>
+          <p class="ts subheading-text">
+            Serving Guanajuato, Mexico<br />street food
+          </p>
+          <p class="story content-text">
+            Family owned and operated business that started in 2002. Specialized
+            in Guanajuato, Mexico home style cooking. Incorporating home taught
+            dishes with a spin. welcoming customers with a freindly smile
+          </p>
+        </div>
+        <img src="img/tacos-truck-pic.jpeg" />
       </div>
     </div>
 
     <!-- image carosuel -->
-  <div id="slider">
```

Comment:
You had a couple of extra divs in here so I got rid of them. You should be able to figure out what's wrong here and fix it now.